views/firstLogin/sections/parent_data.hbs

```diff
@@ -1,24 +1,24 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Ihre Daten</h2>
-  <p>Für Schüler unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahren bedarf ein Erziehungsberechtigter in die Nutzung der {{@root.theme.short_title}} einwilligen.</p>
-  <p>Dazu benötigen wir folgende Angaben von Ihnen:</p>
+  <h2 class="h4">{{$t "firstLogin.parent.heading.yourData" }}</h2>
+  <p>{{$t "firstLogin.parent.global.text.forStudentsUnder" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "firstLogin.parent.text.aParentOrLegalGuardianMystUse" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.toConsent" }}</p>
+  <p>{{$t "firstLogin.parent.text.forThisWeNeedTheFollowingInformationFromYou" }}</p>
   <div class="row">
     <div class="form-group col-md-6 col-sm-12">
-      <label>Vorname: *</label>
+      <label>{{$t "firstLogin.parent.global.label.firstName" }} *</label>
       <input class="form-control linked" name="parent_firstName" type="text" placeholder="Ihren Vornamen hier eingeben" required/>
     </div>
     <div class="form-group col-md-6 col-sm-12">
-      <label>Nachname: *</label>
+      <label>{{$t "firstLogin.parent.global.label.lasttName" }} *</label>
       <input class="form-control linked" name="parent_lastName" type="text" placeholder="Ihren Nachname hier eingeben" required/>
     </div>
   </div>
 
   <div class="form-group">
-    <label>Ihre E-Mail-Adresse * <small>(Darf nicht gleich der E-Mail-Adresse Ihres Kindes sein)</small></label>
+    <label>{{$t "firstLogin.parent.global.label.yourEmailAddress" }} * <small>{{$t "firstLogin.parent.global.label.mustNotBeYourChildsEmailAddress" }}</small></label>
     <input class="form-control" type="email" name="parent_email" required placeholder="eltern@schul-cloud.org" autocomplete />
   </div>
   <p class="explanation nomargin">
-    Wir benötigen eine E-Mail-Adresse, damit Ihre Einwilligung in die Nutzung der {{@root.theme.short_title}} rechtlich gültig ist. Die {{@root.theme.short_title}} gibt die Adresse nicht weiter. Sie erhalten keine unerwünschten Mails.
+    {{$t "firstLogin.parent.text.weNeedAnEmailAddress" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.islegallyValidThe" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.doesNotPassOnTheAddress" }}
     <br/>
     Zusätzlich planen wir die Entwicklung eines Eltern-Zugangs, über den Sie mit dem Klassenlehrer kommunizieren und mit dem Sie die Lernfortschritte Ihres Kindes nachvollziehen können. Sobald diese Funktionen zur Verfügung stehen, fragen wir Sie, ob wir Ihre E-Mail-Adresse dafür verwenden dürfen.
```

Comment:
Missing translation

Reply:
why are you resolving issues here, which are clearly not resolved? please fix